NuGet.Docs/Create/Creating-and-Publishing-a-Package.md

```diff
@@ -128,6 +128,32 @@ For a detailed walkthrough showing how to create and publish NuGet packages from
 see [The easy way to publish NuGet packages with sources](http://blog.davidebbo.com/2011/04/easy-way-to-publish-nuget-packages-with.html) 
 on David Ebbo&#8217;s blog.
 
+### At Build Time
+
+It may be advantageous for you to enhance your project build process to include the ability to generate a NuGet package at the completion of a successful build.  You can augment your build script to include the steps necessary to assembly your package once the compilation has completed by adding an AfterBuild target as follows, assuming that you have the nuget.exe utility available in the path:
+
+	<Target Name="AfterBuild" Condition=" '$(Configuration)' == 'Release'">
+		
+		<Exec Command="nuget pack MyProject.csproj -Prop Configuration=Release"></Exec>
```

Comment:
add a note that pack .csproj is not fully featured, and works for many simple cases, for more complex cases you want to pack to a nuspec